css/utilities/padding.css

```diff
@@ -0,0 +1,7 @@
+.padding-p5-1p5 {
+  padding: 0.5rem 1.5rem;
+}
+
+.padding-1p2-2 {
+  padding: 1.2rem 2rem;
+}
```

Comment:
These are nice. But very specific. and since padding and margins need to be consistent across, you can define this can name it like  - `padding-md, padding-sm, padding-lg, padding-2xl` etc. See what's very handy.

Reply:
Actually, I use naming like this so that we can understand what value is getting applied. Like padding `1.2` for top and bottom, and `2` for left and right in `padding-1p2-2`.

And If I will name them as `padding-md` then I will have to check what value is getting applied to get my work done.

Reply:
Valid point. But like in a real scenario, if someone else is using your library, they will have to first understand your naming convention. 

Also, if you see. - why do we need rems in `1.2` ? why can't it be very 1 or 1.5? What I mean is, you set a standard across your library that the minimum padding will in multiples of `0.5` .

Reply:
This is a nice thing, that we should be consistent with multiples of padding too. Thank you for sharing this, will keep this in mind.